packages/server/postgres/migrations/1698263155574_addPrioritizationTemplates.ts

```diff
@@ -0,0 +1,212 @@
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+import {Client} from 'pg'
+import {r} from 'rethinkdb-ts'
+import connectRethinkDB from '../../database/connectRethinkDB'
+import TemplateDimension from '../../database/types/TemplateDimension'
+import TemplateScale from '../../database/types/TemplateScale'
+import getPgConfig from '../getPgConfig'
+import getPgp from '../getPgp'
+
+const createdAt = new Date()
+
+interface ScaleValue {
+  color: string
+  label: string
+}
+
+interface Scale {
+  id: string
+  name: string
+  values: ScaleValue[]
+}
+
+interface Dimension {
+  id: string
+  name: string
+  scaleId: string
+}
+
+interface Template {
+  id: string
+  name: string
+  dimensions: Dimension[]
+}
+
+const getTemplateIllustrationUrl = (filename: string) => {
+  const cdnType = process.env.FILE_STORE_PROVIDER
+  const partialPath = `Organization/aGhostOrg/template/${filename}`
+  if (cdnType === 'local') {
+    return `/self-hosted/${partialPath}`
+  } else if (cdnType === 's3') {
+    const {CDN_BASE_URL} = process.env
+    if (!CDN_BASE_URL) throw new Error('Missng Env: CDN_BASE_URL')
+    const hostPath = CDN_BASE_URL.replace(/^\/+/, '')
+    return `https://${hostPath}/store/${partialPath}`
+  }
+  throw new Error('Mssing Env: FILE_STORE_PROVIDER')
+}
+
+const MOSCOW_SCALE_CONFIG: TemplateScale = {
+  createdAt,
+  id: 'moscowScale',
+  isStarter: true,
+  name: 'MoSCoW',
+  sortOrder: 4,
+  teamId: 'aGhostTeam',
+  updatedAt: createdAt,
+  values: [
+    {label: 'M', color: PALETTE.TOMATO_500},
+    {label: 'S', color: PALETTE.TERRA_300},
+    {label: 'C', color: PALETTE.GOLD_300},
+    {label: 'W', color: PALETTE.GRASS_300},
+    {label: '?', color: PALETTE.ROSE_500},
+    {label: 'Pass', color: PALETTE.GRAPE_500}
+  ]
+}
+
+const NEW_TEMPLATE_CONFIGS: Template[] = [
+  {
+    id: 'moscowPrioritizationTemplate',
+    name: 'MoSCoW Prioritization',
+    dimensions: [
+      {
+        id: 'moscowPriorityDimension',
+        name: 'Priority? Must/Should/Could/Won’t Have',
```

Comment:
+1 Most dimension names generally seem to be a simpler label than this. I'd suggest renaming this to just "Priority", since the "MoSCoW" scale label is shown right next to the dimension label in the product:
<img width="633" alt="Screen Shot 2023-10-31 at 1 36 02 PM" src="https://github.com/ParabolInc/parabol/assets/9013217/dd11e023-9b49-40b9-9272-9176759faa11">

Reply:
I agree that I’m stretching the use of this label. However there’s not a place in the template where I can specify what MoSCoW means. I assume most of the time folks reaching for this are familiar with it. However participants may not be as familiar and have to remember ‘what is S again’? Also, we have a good number of folks whose first language isn’t English. This might help there, erring on the side of clarity over brevity. In the future, I imagine we expand on Poker, Dimensions, Scales etc. to support more interaction types and better views into labels, values, prompts, etc. and I wouldn’t abuse this label. Maybe even tooltips with a label e.g. ‘Must Have’ when hovering the cards would help in a case like this. In the meantime it fits in the view, even mobile is okay. Thanks for reviewing!
<img width="420" alt="Screenshot 2023-11-01 at 11 54 14" src="https://github.com/ParabolInc/parabol/assets/307286/3e497e3f-094b-46a7-aeb8-d30be1142424">